### PR TITLE
Taxi tribute Vault

### PR DESCRIPTION
**Pull request by Probe1**
## Ready for review/merge.

Video

https://youtu.be/4aiC90JRyqk?t=72

Screenshot

dreammaker_2016-12-19_02-36-40

### CODE REVIEW

**Review by PJB3005 — APPROVED** [review]

**Comment by 9600bauds** [comment]
Why is some of it powered?

**Comment by Probe1** [comment]
the real question is why isn't all of it powered.

the answer to that is I put it all under one area and there are about 10 APCs so stupid shit happened.  that's something I plan to clean up tomorrow when I have the energy.

**Comment by 9600bauds** [comment]
hehahehaheuae get it energy

**Comment by Ephx1** [comment]
I think the area is a little too clean cut for some sort of disaster. Like it was sliced straight out of the map instead of blown up.

**Comment by Probe1** [comment]
Yeah I was getting that vibe too.  Rather than really fuck up engineering I think I'll go back and make it a tiny bit bigger and add in a bit more of the station outside it.

**Comment by yclatious** [comment]
Hold on, this as a vault?

You want to make who do a quick space trip get a place with a extra shard, 2 gas miners, another pair of rare galoshes, another bartender shotty that you cant get by any normal means, a ton of boards, what looks to be free CE stuff and a singulo invoking thingamajig?

Maybe its just me, but that seems kind of overkill.

**Comment by 9600bauds** [comment]
DON'T FORGET AN EXTRA POLY

**Comment by Probe1** [comment]
Lest he be forgotten.
(He dies immediately)

**Comment by yclatious** [comment]
Cant you use the miner injector to get him alive again?

**Comment by Probe1** [comment]
Hell yes you can.

**Comment by Probe1** [comment]
Also I'm just throwing this out there I've been up for close to 24 hours now and I'm going to probably likely almost certainly address all the balance stuff.

I'm a little disappointed you were unamused by the galoshes and mop next to remains however.

**Comment by NigglyWiggly** [comment]
Eh
I like it but I'd say maybe remove some of the more valuable stuff, make it look like it's already been picked over by scavengers and stuff

**Comment by yclatious** [comment]
Its funny and I did chuckle, but the thing is, it has a lot of rare items and a bit too much loot for a vault, wouldn't you say?

I don't like to be the party pooper, but galoshes, the bartenders shotty, a second poly, that is just too much. Replace the supposed to be rare items with something else, and I don't have much of a issue with it anymore.

**Comment by Probe1** [comment]
@NigglyWiggly I was thinking along the same lines.  I made a note to use a subtype of the hardsuit storage units that are emptied.

@yclatious the labor of traveling through space and finding a vault offsets minor things like a weak shotgun with no ammo when you could have stayed at home and gotten a riot crate with much more ease.  It's not that the bartenders shotgun is valuable.  It's that it's _his._

**Comment by yclatious** [comment]
That's kind of what I mean.

It is the unique bartenders shotgun, it is the unique pair of galoshes, it is the unique Poly the Parrot, and you are just not making them unique anymore. It isn't about effectiveness, you can be infinitely more effective without them, but I just don't see why these items should be devalued for the sake of a small humorous lil item placement gag, stating that the office of the CE is there when the command door already says so, or giving the shotty when you know that clearly is the bartenders backroom by the still alone. Maybe im just nitpicking, but devauling rare items and making Spacepod Explorer Grey Shit get suposedly unique items just seems wrong.

**Comment by kol1th** [comment]
image
triggered

**Comment by NigglyWiggly** [comment]
Having unique items not be unique is typically a bad thing.

But I kind of like the implication of it in this case.
It's another station with an exact copy of your pet parrot and you have no idea how it got there or why.

**Comment by JustSumBody** [comment]
DEEPEST
L O R E
O
R
E

**Comment by Kurfursten** [comment]
I liked how comfy small Taxi engineering was too but it seems a little weird that the singulo wasn't involved in eating the station, since that's how 90% of Taxi rounds ended. Realistically a Taxi tribute value would be the opposite of this map, a civstation with the engineering chunk missing. 

You could even store some fun cargo in the remains of the cargo bay.

**Comment by Probe1** [comment]
I just spent 90 minutes trying to find out why all the apcs were being called duplicates and deleted.

I tried everything I could think of until I realized dreammaker automatically checked the new file when it was created.  The game was spawning the dmm somewhere.

I hate dreammaker. I really do.

**Comment by Probe1** [comment]
@Kurfursten I kinda feel the same way but this is what I was asked for and I'll deliver it.  There could be another taxi vault /shrug

**Comment by Probe1** [comment]
The screenshot in the OP has been updated.

@yclatious galoshes turned into ruined subtype thanks to @NigglyWiggly 's sprite.
asf .  They are no longer no_slip.  Shotgun removed.

@Ephx1 I sliced off the bar entirely and some of the pod bay to give it more jaggedness.  I need to maintain the cohesion and rebuild quality of engineering so I really can't go ham on it.  I hope this is acceptable.

@Kurfursten same reason as above for why I didn't just carve a big fucking singlo path through it.  That's how we remember it but it wouldn't be very fun to rebuild if 90% of the important stuffs missing.

File: code/modules/clothing/shoes/miscellaneous.dm [hunk 1]
```diff
@@ -106,15 +106,20 @@
...
 	name = "galoshes"
+	desc = "Rubber boots"
 	icon_state = "galoshes"
 	permeability_coefficient = 0.05
 	flags = NOSLIP
 	slowdown = SHOES_SLOWDOWN+1
 	species_fit = list(VOX_SHAPED)
 	heat_conductivity = INS_SHOE_HEAT_CONDUCTIVITY
 
+/obj/item/clothing/shoes/galoshes/vault/broken
```
Comment (Shadowmech88) [inline code/modules/clothing/shoes/miscellaneous.dm]: Why not just `/obj/item/clothing/shoes/galoshes/broken`?

Comment (Probe1) [reply code/modules/clothing/shoes/miscellaneous.dm]: So people know what the fuck they're used for.  I'm torn honestly.  I want to put it in the vault file but i hate jamming stuff into random files and making it hard to find.  I can drop the vault path if you think its unneeded.

Comment (Shadowmech88) [reply code/modules/clothing/shoes/miscellaneous.dm]: What if someone wants to put them somewhere that *isn't* a vault at some point?

People don't need to know that these have been placed in a vault. All they need to know is that they're a child of galoshes with a different sprite and the slip resistance removed. They can be used for whatever anyone wants to use them for.

Comment (Probe1) [reply code/modules/clothing/shoes/miscellaneous.dm]: Can't argue with that.

**Comment by Probe1** [comment]
Alright I think this is done.

**Pull request merged**